python/llm/src/ipex_llm/transformers/npu_models/convert_mp.py

```diff
@@ -85,9 +87,29 @@ def optimize_llm_pre(model: torch.nn.Module, qtype):
 
     if model.config.model_type == "qwen2":
         from ipex_llm.transformers.npu_models.qwen2_mp import split_mlp_down_proj
-        from ipex_llm.transformers.npu_models.qwen2_mp import split_mlp_forward
         model.apply(split_mlp_down_proj)
 
+        # for Qwen2-7B-Insturct, divide lm_head into 7 parts
+        if model.config.hidden_size == 3584 and model.config.vocab_size == 152064 and \
```

Comment:
IPEX_LLM_CPU_LM_HEAD==0?

Reply:
it's the same as `cpu_lm_head` condition

Reply:
Do we need to split when IPEX_LLM_CPU_LM_HEAD is true?

Reply:
I feel it's not necessary to split for IPEX_LLM_CPU_LM_HEAD is true as it may just increase onverhead?

Reply:
By the way, during my test, I found for current main branch, Qwen2 with IPEX_LLM_CPU_LM_HEAD=1 will have strange output:
![image](https://github.com/user-attachments/assets/06a34fb5-3329-40e3-aa21-dfc3432c74d6)